### PR TITLE
New API /api/friendica/profile/show

### PR DESCRIPTION
**Pull request by gerhard6380**
Returns to caller all data of all or a single profile of the autenticated user, XML output works, but needs to be checked by someone with more XML experience.

### CODE REVIEW

**Comment by rabuzarus** [comment]
Because I can't comment inline because github says api.php does have to much changes I will do it here.

1.) please use tabs (not spaces) for indents
2.) both new functions does have some wrong indents
3.) in the doxygen header of `function api_friendica_profile_show()` there is a line with ` @param string $profile_id optional parameter to provide the id of the profile to be returned` but this function does only use `$type` as input variable (`function api_friendica_profile_show($type)`). So this line can be removed

**Comment by annando** [comment]
It seems as if your text editor had changed had done changes to every line. That's really bad for the tracking of changes. Could you check what your file editor did wrong and correct it? I wouldn't like to accept the pull request in this state since it is a potential cause for merge problems.

**Comment by gerhard6380** [comment]
CRLF issue corrected, sorry I expected that Github would manage this automatically. I have also changed the behavior of Visual Studio with the indents from spaces to tabstops. 
doxygen header: I have removed the comment, the optional parameter profile_id is documented in the api.md file

File: include/api.php [hunk 1]
```diff
@@ -3880,6 +3933,72 @@ function api_friendica_direct_messages_search($type){
 	api_register_func('api/friendica/direct_messages_search', 'api_friendica_direct_messages_search', true);
 
 
+	/**
+	 * @brief return data of all the profiles a user has to the client
+	 *
+	 * @param string $profile_id optional parameter to provide the id of the profile to be returned
```
Comment (annando) [inline include/api.php]: The function does have a single parameter - but there are two described in the doxygen header. Could you correct it?

Comment (gerhard6380) [inline include/api.php]: done

**Pull request merged**